niceman/support/starcluster/sshutils.py

```diff
@@ -236,7 +236,7 @@ def load_remote_rsa_key(self, remote_filename):
         rfile.close()
         return key
 
-    def makedirs(self, path, mode="0755"):
+    def makedirs(self, path, mode=0o755):
```

Comment:
is it "identically" treated? or there is a bug in starcluster which needs to be fixed "uptream"?